integration-cli/docker_cli_history_test.go

```diff
@@ -54,10 +53,7 @@ RUN echo "Z"`,
 	for i := 0; i < 26; i++ {
 		echoValue := fmt.Sprintf("echo \"%s\"", expectedValues[i])
 		actualValue := actualValues[i]
-
-		if !strings.Contains(actualValue, echoValue) {
-			c.Fatalf("Expected layer \"%s\", but was: %s", expectedValues[i], actualValue)
-		}
+		c.Assert(actualValue, checker.Contains, echoValue, check.Commentf("Expected layer \"%s\", but was: %s", expectedValues[i], actualValue))
```

Comment:
I looked at the output when it fails and I agree the comment really isn't needed, so I removed it.